src/Pact/Types/SizeOf.hs

```diff
@@ -143,6 +144,13 @@ instance SizeOf BS.ByteString where
       byteStringSize = (9 * wordSize) + byteStringLength
       byteStringLength = fromIntegral (BS.length bs)
 
+instance SizeOf SBS.ShortByteString where
+  sizeOf _ bs = byteStringSize
+    where
+      byteStringSize = (9 * wordSize) + byteStringLength
+      -- byteStringLength = fromIntegral (SBS.length bs)
+      byteStringLength = fromIntegral (BS.length $ SBS.fromShort bs)
```

Comment:
Considering that `ByteString` length can be fairly large in the size of bytes, this makes sense to do, though, I doubt it's a bottleneck. I'm fine with versioning to get this behavior nailed down sooner rather than later imo.

Reply:
Perf on large `ByteString`s is probably pretty bad. `length` (https://hackage.haskell.org/package/utf8-string-1.0.2/docs/src/Data.ByteString.UTF8.html#length) is calling `decode` (https://hackage.haskell.org/package/utf8-string-1.0.2/docs/src/Data.ByteString.UTF8.html#decode) on each individual byte.

So, it really depends how often it is called and how large the values are.